tc/core/mapping_options-inl.h

```diff
@@ -161,7 +161,7 @@ MappingOptionsView& MappingOptionsView::scheduleFusionStrategy(
 
 MappingOptionsView& MappingOptionsView::scheduleFusionStrategy(
     const std::string& str) {
-  FusionStrategy fs;
```

Comment:
Who complains about this and why?

Reply:
see #500, I did not understand that either.

Reply:
see #504 (it should show in the CI)